index.html

```diff
@@ -112,140 +133,148 @@ <h3>Multi-Post Stories</h3>
               </div>
             </div>
 
-            <div class="work-2">
-              <div class="work-detail">
-                <div class="main-detail">
-                  <h4>professional art printing data</h4>
-                  <p class="paragraph">
-                    A daily selection of privately personalized reads; no
-                    accounts or sign-ups required. has been the industry's
-                    standard
-                  </p>
-                </div>
-                <div class="work-tools work-stack">
-                  <ul>
-                    <li>html</li>
-                    <li>bootstrap</li>
-                    <li>Ruby</li>
-                  </ul>
-                </div>
-                <div class="see-project">
-                  <button class="large-button">see project</button>
+            <div class="all-works__detail">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please `style` each work `block` as shown in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=kLoX8FBKAxK9lEJJ-0) and add correct `Hover effect` to each work block, I have `highlighted` them below, please correct them 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/220960656-7887e375-04cc-4089-b120-d4f34f20e1bb.png)